src/cmd_line.rs

```diff
@@ -132,6 +150,14 @@ pub fn handle_command_line_arguments() -> Result<(), String> {
         wsl: matches.is_present("wsl"),
         frameless: matches.is_present("frameless") || std::env::var("NEOVIDE_FRAMELESS").is_ok(),
         geometry: parse_window_geometry(matches.value_of("geometry").map(|i| i.to_owned()))?,
+        wayland_app_id: matches
+            .value_of("wayland_app_id")
+            .unwrap_or_default()
```

Comment:
```suggestion
            .unwrap_or(String::from("neovide"))
```

Make sure default value is "neovide", lowercase. As by default it matches executable name.